fulfillment-service/proto/private/osac/private/v1/hub_type.proto

```
@@ -16,6 +16,9 @@ syntax = "proto3";
 package osac.private.v1;
 
 import "osac/private/v1/metadata_type.proto";
+import "cleanapi/v1/cleanapi.proto";
```

Buf (1.72.0)

[error] imported file does not exist

(COMPILE)